R/iSEE-main.R

```diff
@@ -1414,7 +1414,12 @@ iSEE <- function(
                 if (enc$Type=="rowStatTable") {
                     incoming <- input[[paste0(enc$Type, enc$ID, "_rows_all")]]
                 } else {
-                    incoming <- which(.get_selected_points(rownames(gene_data), origin, pObjects$memory, pObjects$coordinates))
+                  selected <- .get_selected_points(rownames(gene_data), origin, pObjects$memory, pObjects$coordinates)
+                  if (is.null(selected)) {
+                    showNotification("Invalid: empty selection", type="warning")
+                    return(NULL) # avoid corner case: which(NULL)
```

Comment:
Are you missing an `incoming <- which(selected)`?

Reply:
Four spaces; respect the adjacent coding style unless you're willing to change it for the entire function.

Reply:
Not that two spaces is a good idea. I can barely see the indents with 2.